aptos-move/framework/aptos-stdlib/sources/cryptography/bls12381.move

```diff
@@ -516,7 +516,7 @@ module aptos_std::bls12381 {
     }
 
     #[test]
-    #[expected_failure(abort_code = 65537)]
+    #[expected_failure(abort_code = 65537, location = Self)]
```

Comment:
Is this `location` field now mandatory with `expected_failure`? I couldn't find it in the Move book chapter on unit testing: https://move-language.github.io/move/unit-testing.html

Reply:
Good question. This is a new warning that was added in https://github.com/move-language/move/commit/926c4a1df234a373c4f11929f69be76c9581ea74. The compiler will raise warnings if there isn't a location field, which is fine by itself. However the unit testing framework currently treats warnings as test failures, which is not the desired behavior and we need to fix that separately.